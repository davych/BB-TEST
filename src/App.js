import './App.css';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';
function App() {
  return (
    <div className="App">
      <Typography variant="h3">Responsive h3</Typography>
      <Button variant="contained" color="primary">
        你好，世界
      </Button>
    </div>
  );
}

export default App;
