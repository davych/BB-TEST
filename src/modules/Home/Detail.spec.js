import { render, screen } from '@testing-library/react';
import App from './Detail';

test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/1/i);
  expect(linkElement).toBeInTheDocument();
});
