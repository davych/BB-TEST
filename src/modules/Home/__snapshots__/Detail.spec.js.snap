// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home detail 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-6"
    />
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
    />
  </div>
</DocumentFragment>
`;

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-6"
    />
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
    />
  </div>
</DocumentFragment>
`;
